install/installer/pkg/config/v1/config.go

```diff
@@ -201,6 +201,7 @@ type ContainerRegistryExternal struct {
 type S3Storage struct {
 	Bucket      string    `json:"bucket" validate:"required"`
 	Certificate ObjectRef `json:"certificate" validate:"required"`
+	Region      string    `json:"region"`
```

Comment:
Would it make sense to set `cfg.Config.Metadata.Region` as default value (especially for backwards compatibility)?

Reply:
By the way: Couldn't we just set the region of `metadata` instead?

```
metadata:
  region: local
```

What is the reason that we need a specific S3 storage region that differs from the metadata region?

As far as I see this in the code `metadata.region` is for the storage provider and here:

https://github.com/gitpod-io/gitpod/blob/1119e5578bfe502ae99b90708649ee19523c8cc0/install/installer/pkg/common/common.go#L53

Don't know for what the env variable is used, though. Maybe also for the storage provider?

Reply:
I understand what you mean, this probably is true and we don't need a separate `region` field for each s3 storage. In my mind I was very convinced that people might want to use buckets belonging to different regions for different usecases.

Reply:
If the user decides not to configure the object storage and uses Minio in-cluster, this will remain as `local` and will error